sql/sql_show.cc

```diff
@@ -6053,24 +6062,24 @@ static int get_schema_column_record(THD *thd, TABLE_LIST *tables,
     restore_record(table, s->default_values);
 
 #ifndef NO_EMBEDDED_ACCESS_CHECKS
-    uint col_access;
-    check_access(thd,SELECT_ACL, db_name->str,
-                 &tables->grant.privilege, 0, 0, MY_TEST(tables->schema_table));
-    col_access= get_column_grant(thd, &tables->grant,
-                                 db_name->str, table_name->str,
-                                 field->field_name.str) & COL_ACLS;
-    if (!tables->schema_table && !col_access)
-      continue;
-    char *end= tmp;
-    for (uint bitnr=0; col_access ; col_access>>=1,bitnr++)
-    {
-      if (col_access & 1)
+      uint col_access;
```

Comment:
This hunk is indented wrong. Two extra spaces everywhere.

Reply:
Thanks, updated